android/KMEA/app/src/main/java/com/tavultesoft/kmea/data/CloudRepository.java

```diff
@@ -89,6 +91,10 @@ public boolean getCacheValidity(@NonNull Context context) {
     boolean loadResourcesFromCache = this.shouldUseCache(context, CloudDataJsonUtil.getResourcesCacheFile(context));
 
     boolean cacheValid = loadLexicalModelsFromCache && loadResourcesFromCache;
+
+    if (VersionUtils.isLocalBuild() || VersionUtils.isTestBuild()) {
+      return false;
+    }
```

Comment:
This should be at the top of the function with a comment explaining why we are doing this.